setup.py

```diff
@@ -29,7 +29,7 @@
     },
     install_requires=[
         "googleapis-common-protos>=1.57",
-        "flyteidl>=1.5.6",
+        "flyteidl>1.5.6",
```

Comment:
this should be `flyteidl>=1.5.9`, no?

Reply:
bumped version to 1.5.10. we yanked 1.5.9